python/cudf/cudf/utils/gpu_utils.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020-2022, NVIDIA CORPORATION.
+# Copyright (c) 2020-2023, NVIDIA CORPORATION.
```

Comment:
Looks like an extraneous copyright update (probably was automatically inserted during an earlier commit containing changes to this file that were later reverted).